src/app/components/authorized/add-new-subreddit/add-new-subreddit.component.ts

```diff
@@ -15,17 +17,16 @@ export class AddNewSubredditComponent {
         description: '',
     };
 
-    constructor(private _gatewayService: GatewayService) {}
+    errorObject?: Error;
```

Comment:
errorObject does not need the "Object" part as it is redundant. Making this "error" is sufficient. Ill approve the PR, but update it.

Reply:
changed name as requested